Kconfig

```diff
@@ -1170,18 +1169,18 @@ menu "LVGL configuration"
 	menu "Others"
 		config LV_USE_SNAPSHOT
 			bool "Enable API to take snapshot"
-			default y if !LV_CONF_MINIMAL
+			default n if !LV_CONF_MINIMAL
```

Comment:
Why turn off by default?

Reply:
Turned off by default based on https://github.com/lvgl/lvgl/blob/master/lv_conf_template.h#L746

Reply:
OK, let's be consistent with lv_conf_template.